.bazelci/presubmit.yml

```diff
@@ -27,6 +27,7 @@ default_windows_targets: &default_windows_targets
   - "-//bindgen/..."
   - "-//test/proto/..."
   - "-//test/unit/pipelined_compilation/..."
+  - "-//test/cargo_build_script:test_exec_root_access.build.feature_enabled"
```

Comment:
Can you delete this in favor of a `target_compatible_with` flag on the target should you need it?